agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/init/AiConfigCustomizer.java

```diff
@@ -165,7 +165,7 @@ private static void enableInstrumentations(Configuration config, Map<String, Str
     properties.put("otel.instrumentation.log4j-mdc.enabled", "true");
     properties.put("otel.instrumentation.log4j-context-data.enabled", "true");
     properties.put("otel.instrumentation.logback-mdc.enabled", "true");
-    properties.put("otel.instrumentation.methods.enabled", "true");
+    properties.put("otel.instrumentation.ai-methods.enabled", "true");
```

Comment:
how does this relate to https://github.com/microsoft/ApplicationInsights-Java/pull/2513/files#:~:text=super(%22ai%2Dmethods%22)%3B?

Reply:
they have to match